src/pages/QuestionPage.jsx

```diff
@@ -254,26 +47,36 @@ const QuestionPage = () => {
   };
 
   // Update the question status on answer click (either correct or incorrect)
```

Comment:
const renderExplanation = (explanation) => {
    if (!explanation || !Array.isArray(explanation)) return null;
    return explanation.map((sentence, index) => {
      // Split each sentence at commas and map them to new lines without gaps
      const splitSentences = sentence.split(",").map((part, i) => (
        <span key={`${index}-${i}`}>
          {part.trim()}
          {i !== sentence.split(",").length - 1 && <br />}{" "}
          {/* Adds line break except for the last part */}
        </span>
      ));
      return <div key={index}>{splitSentences}</div>;
    });
    put this in a utils file

Reply:
we decided to pass on this